apps/frontend/src/app/components/user-search/user-search-result.component.html

```diff
@@ -1,9 +1,21 @@
 <div class="user-search-result">
   <img class="user-search-result__avatar" [src]="user.avatarURL" />
   <div class="user-search-result__content">
-    <div class="content__alias">{{ user.alias }}</div>
-    <a class="content__steamid" href="http://steamcommunity.com/profiles/{{ user.steamID }}">
-      {{ user.steamID }}
-    </a>
+    <div class="content__alias">
+      <span>{{ user.alias }}</span>
+      <img *ngIf="userRoles.isVerified" class="ml-2 verified" ngSrc="assets/images/badges/BadgeVerified.svg"
+        nbPopover="Verified" nbPopoverTrigger="hint" nbPopoverPlacement="bottom" width="12" height="12" />
+      <img *ngIf="userRoles.isMapper" class="ml-1 mapper" ngSrc="assets/images/badges/BadgeMapper.svg"
+        nbPopover="Mapper" nbPopoverTrigger="hint" nbPopoverPlacement="bottom" width="12" height="12" />
+      <img *ngIf="userRoles.isMod" class="ml-1 mod" ngSrc="assets/images/badges/BadgeMod.svg" nbPopover="Moderator"
+        nbPopoverTrigger="hint" nbPopoverPlacement="bottom" width="12" height="12" />
+      <img *ngIf="userRoles.isAdmin" class="ml-1 admin" ngSrc="assets/images/badges/BadgeAdmin.svg" nbPopover="Admin"
+        nbPopoverTrigger="hint" nbPopoverPlacement="bottom" width="12" height="12" />
+    </div>
+    <div>
+      <a click-stop-propagation class="content__steamid" href="http://steamcommunity.com/profiles/{{ user.steamID }}">
```

Comment:
Probably want this in a new tab, so let's add a `target=_blank`

Reply:
That's added 👍🏼